sample/ERC000038.json

```diff
@@ -0,0 +1,589 @@
+              "value"
+            ],
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "adductor weight": {
+          "minItems": 1,
+          "items": {
+            "required": [
+              "value",
+              "units"
+            ],
+            "properties": {
+              "value": {
+                "type": "string",
+                "pattern": "(0|((0\\.)|([1-9][0-9]*\\.?))[0-9]*)([Ee][+-]?[0-9]+)?"
+              },
+              "units": {
+                "enum": [
+                  "g"
+                ]
+              }
+            }
+          },
+          "description": "Total weight of striated muscle and smooth muscle",
+          "type": "array",
+          "maxItems": 1
+        },
+        "Sampling Campaign": {
+          "type": "array",
+          "maxItems": 1,
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "value"
+            ]
+          },
+          "description": "Refers to a finite or indefinite activity aiming at collecting data/samples, e.g. a cruise, a time series, a mesocosm experiment. Example: TARA_20110401Z.",
+          "minItems": 1
+        },
+        "sample disease status": {
+          "type": "array",
+          "maxItems": 1,
+          "items": {
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            },
+            "required": [
+              "value"
+            ]
+          },
+          "description": "list of diseases with which the subject has been diagnosed at the time of sample collection; can include multiple diagnoses; the value of the field depends on subject; e.g. Charcoal rot (Macrophomina phaseolina), Late wilt (Cephalosporium maydis)",
+          "minItems": 1
+        },
+        "age": {
+          "type": "array",
+          "maxItems": 1,
+          "items": {
+            "required": [
+              "value"
+            ],
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          },
+          "description": "Age of the organism the sample was derived from.",
+          "minItems": 1
+        },
+        "chemical compound": {
+          "type": "array",
+          "items": {
+            "required": [
+              "value"
+            ],
+            "properties": {
+              "value": {
+                "type": "string"
+              }
+            }
+          },
+          "description": "A drug, solvent, chemical, etc., with a property that can be measured such as concentration (http://purl.obolibrary.org/obo/CHEBI_37577).",
+          "minItems": 1
+        },
+        "Depth": {
+          "minItems": 1,
+          "type": "array",
+          "maxItems": 1,
```

Comment:
The structure of an attribute is always the same and can be defined just once, see [here](https://github.com/EMBL-EBI-SUBS/validation-schemas/blob/master/sample/sample-schema.json#L42), this avoids duplication and is less error prone.
Any further customisation, if for example an attribute also requires `"units"` and/or `"terms"` can be done directly on that attribute.